torchvision/transforms/functional_tensor.py

```diff
@@ -217,5 +231,6 @@ def ten_crop(img, size, vertical_flip=False):
 
 
 def _blend(img1, img2, ratio):
-    bound = 1 if img1.dtype.is_floating_point else 255
+    # type: (Tensor, Tensor, float) -> Tensor
+    bound = 1 if img1.dtype == torch.float else 255
```

Comment:
Can you do instead something like `img1.dtype in [torch.half, torch.float32, torch.float64]`?